apps/homepage2/pages/styles.css

```
@@ -1,4 +1,4 @@
-@import url(//fonts.googleapis.com/css?family=Open+Sans:400,700,300,600,800);
+@import url(//fonts.googleapis.com/css?family=Open+Sans:400,700,300,600,800&display=swap);
```

Stylelint (17.12.0)

[error] Expected "url(//fonts.googleapis.com/css?family=Open+Sans:400,700,300,600,800&display=swap)" to be ""//fonts.googleapis.com/css?family=Open+Sans:400,700,300,600,800&display=swap"" (import-notation)

(import-notation)